include/osquery/json_parser.h

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
Can you move this to `./osquery/core/json.h`? I am resisting adding new files to `./include`.